x-pack/test_serverless/functional/test_suites/search/config.feature_flags.ts

```diff
@@ -19,9 +19,11 @@ export default createTestConfig({
   suiteTags: { exclude: ['skipSvlSearch'] },
   // add feature flags
   kbnServerArgs: [
-    `--xpack.security.roleManagementEnabled=true`,
+    `--xpack.cloud.id='ES3_FTR_TESTS:ZmFrZS1kb21haW4uY2xkLmVsc3RjLmNvJGZha2Vwcm9qZWN0aWQuZXMkZmFrZXByb2plY3RpZC5rYg=='`,
```

Comment:
Are these changes supposed to be in here?

Reply:
yes, we need a cloud id to decode the es url to test the flyout in FTR tests